scripts/lint.sh

```diff
@@ -3,9 +3,11 @@ set -euxo pipefail
 
 poetry run cruft check
 poetry run mypy -p isort -p tests
-poetry run black --target-version py38 --check .
+poetry run black --target-version py38 .
```

Comment:
This fundamentally changes how the lint script works. I recommend adding `--check` back in to maintain its behavior.

Reply:
> This fundamentally changes how the lint script works. I recommend adding `--check` back in to maintain its behavior.

whoops, I removed this to easily re-format everything with the new `black` version, but it shouldn't have been committed, adding the flag back: 5760cc9